doc/api/v0.yaml

```diff
@@ -224,15 +219,61 @@ components:
           format: date-time
         chain_voteplan_payload:
           type: string
-        fund_id:
-          type: integer
-          format: int32
+
+    ChallengeType:
+      type: string
+      enum:
+        - simple
+        - community-choice
+
+    ProposalWithChallengeInfo:
+      discriminator:
+        propertyName: challenge_type
+        mapping:
+          simple: SimpleProposal
+          community-choice: CommunityChoiceProposal
+      allOf:
+        - $ref: "#/components/schemas/Proposal"
+        - type: object
+          properties:
+            fund_id:
+              type: integer
+              format: int32
+            challenge_id:
+              type: integer
+              format: int32
+            challenge_type:
+              $ref: "#/components/schemas/ChallengeType"
+
+    SimpleProposal:
+      allOf:
+        - $ref: "#/components/schemas/ProposalWithChallengeInfo"
+        - type: object
+          properties:
+            proposal_solution:
+              type: string
+
+    CommunityChoiceProposal:
+      allOf:
+        - $ref: "#/components/schemas/ProposalWithChallengeInfo"
+        - type: object
+          properties:
+            proposal_brief:
+              type: string
+            proposal_importance:
+              type: string
+            proposal_goal:
+              type: string
+            proposal_metrics:
```

Comment:
Is this also some textual description?

Reply:
Yes, just another field describing the key metrics.
The cue is that all of these fields need proper OpenAPI descriptions.